build.gradle

```diff
@@ -24,22 +24,264 @@
  * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
  * POSSIBILITY OF SUCH DAMAGE.
  */
+//use new plugins method
+plugins {
+    id 'net.saliman.cobertura' version '3.0.0' apply false
+    id 'org.ajoberstar.git-publish' version '3.0.0' apply false
+    id 'nebula.release' version '15.2.0' apply false
+}
+
 description = 'EMC Object Client for Java - provides REST access to object data on EMC platforms using the Atmos and S3 APIs.'
 
 ext.githubProjectName = 'ecs-object-client-java'
 
-buildscript {
-    ext.commonBuildVersion = '1.6'
-    ext.commonBuildDir = "https://raw.githubusercontent.com/EMCECS/ecs-common-build/v$commonBuildVersion"
-    apply from: "$commonBuildDir/ecs-publish.buildscript.gradle", to: buildscript
-}
-
 allprojects {
-    apply from: "$commonBuildDir/ecs-publish.gradle"
+    group 'com.emc.ecs'
+
```

Comment:
Move the plug-in block (below) to here and then close the allProjects block.. I don’t think any of the rest of this block is necessary for the geopinning tool.

Reply:
Might also need to move the tools configuration here too

Reply:
allproject now only contains: param setting, plugin apply block, configurations and repositories